satellite/repair/repair_test.go

```diff
@@ -273,8 +278,8 @@ func TestCorruptDataRepair(t *testing.T) {
 		err = corruptedNode.Storage2.BlobsCache.Delete(ctx, blobRef)
 		require.NoError(t, err)
 
-		// corrupt data and write back to storagenode
-		pieceData[0]++ // if we don't do this, this test should fail
+		// corrupt piece data and write back to storagenode
+		pieceData[pieceSize-1]++ // if we don't do this, this test should fail
```

Comment:
Why change this from `0` to `pieceSize-1`? Should be fine, but I am just curious

Reply:
The first couple bytes of the piece are actually for `FormatVersion` which caused the download to fail before it actually got to the verification part of the code. So I changed it to modify the actual piece data so it will fail at the verification part. Maybe I should add some comment on that